tljh/user_creating_spawner.py

```diff
@@ -32,3 +33,8 @@ def start(self):
                     user.ensure_user_group(system_username, group)
         return super().start()
 
+cfg = configurer.load_config()
+if cfg['services']['configurator']['enabled']:
```

Comment:
Very cool type-mashing! Can you leave a comment here on why we are doing this, and how it works?